src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/tests/Dropdown.test.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import {mount, render} from 'enzyme';
+import {fireEvent, render, screen} from '@testing-library/react';
```

Comment:
would be nice to use the `@testing-library/user-event` package instead of `fireEvent` 🙂 see https://github.com/sulu/sulu/pull/6783 for some examples